portkey_ai/api_resources/apis/generation.py

```diff
@@ -84,6 +84,41 @@ def __init__(self, client: APIClient) -> None:
         super().__init__(client)
         self.completions = Completions(client)
 
+    def render(
+        self,
+        *,
+        prompt_id: str,
+        variables: Optional[Mapping[str, Any]] = None,
+        config: Optional[Union[Mapping, str]] = None,
```

Comment:
config is not required for render method. Please remove this and other config references in this method

Reply:
Same for async method as well.